time sent_time
bool constant_speed
bool accelerate
bool decelerate
bool change_lane
bool reset_run
int32 id
